src/components/IdScanner/IdScanner.tsx

```diff
@@ -142,7 +143,9 @@ export const IdScanner: FunctionComponent<IdScanner> = ({
   hasLimitedInterestArea = true,
 }) => {
   const [platform] = useState<string>(Platform.OS);
-  const [hasCameraPermission, setHasCameraPermission] = useState(false);
+  const [hasCameraPermission, setHasCameraPermission] = useState(
+    process.env.JEST_WORKER_ID ? true : false
```

Comment:
might be good to add a comment here as well so that it's clear why this is required!

Reply:
ehh ps, can add an explanation on why that is needed for jest tests? or maybe a link explaining this too!

Reply:
removed setting hasCameraPermission as true in default for Jest environment and mocked the Camera.requestPermissionsAsync() instead.